config/Shared/config_default.php

```diff
@@ -674,6 +676,9 @@
     $config[KernelConstants::DOMAIN_WHITELIST],
     $aopApplicationConfiguration['APP_DOMAINS'] ?? [],
 );
+
+$config[KernelConstants::DOMAIN_WHITELIST][] = '*.bazaarvoice.com';
```

Comment:
I think this one is controversial, since it's not feasible to do this for every integration we do.
The domain whitelist is already covered in the integration docs, so I think we should just remove this one: https://docs.spryker.com/docs/pbc/all/ratings-reviews/202212.0/third-party-integrations/integrate-bazaarvoice.html#add-the-bazaarvoice-domain-to-your-allowlist

@matweew thoughts?

Reply:
no, demoshop is also integration the same as our customers can do on their projects

let's leave.